Formula/dtm.rb

```diff
@@ -0,0 +1,26 @@
+class Dtm < Formula
+  desc "Cross-language distributed transaction manager"
+  homepage "http://d.dtm.pub"
+  url "https://github.com/dtm-labs/dtm/archive/refs/tags/v1.8.1.tar.gz"
+  sha256 "79ec193ad5dd28a6bf01290df7e457bc1b0316bc3f153f9f9bafe7daa612d6cf"
+  license "BSD-3-Clause"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X main.Version=v#{version}"), "-o", bin/"dtm"
+    system "go", "build", *std_go_args(ldflags: "-s -w"), "-o", bin/"dtm-qs", "qs/main.go"
```

Comment:
The first one can use the default (which is the formula name), the second one can use an argument I forgot we had.
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w -X main.Version=v#{version}")
    system "go", "build", *std_go_args(ldflags: "-s -w", output: bin/"dtm-qs"), "qs/main.go"
```